src/content/sessions/2023-09-01.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Rider: CE custom operations completion list"
+preview: ""
+isDraft: true
+date: 2023-09-01T13:00:00.000Z
+slug: "2023/09/01"
+champion: Jindřich Ivánek
+zoomLink: ""
+zoomPasscode: ""
+issueLink: "https://youtrack.jetbrains.com/issue/RIDER-15832/Put-custom-operations-at-the-top-of-completion-list-if-its-invoked-on-top-level-inside-computation-expression"
+company: ""
```

Comment:
This can be `Ciklum` I think, see https://github.com/amplifying-fsharp/sessions/issues/25

Reply:
Good spot. Thanks